pkg/config/tlscfg/cert_watcher_test.go

```diff
@@ -340,83 +326,6 @@ func createTimestampDir(t *testing.T, dir string, ca, cert, key string) {
 	require.NoError(t, err)
 }
 
-func TestAddCertsToWatch_err(t *testing.T) {
```

Comment:
why delete this test? I think it could've helped the coverage I commented on earlier.

Reply:
I thought invalid file names were already tested in `fswatcher`, so there's no need to test it again in `static_handler` and `cert_watcher`?

Reply:
there are different levels of error handling, the test here would exercise error code paths that only exist in cert_watcher